### DISCUSSION TITLE
What is the difference between `observable.map` and proxied observable?

### DISCUSSION THREAD

**Discussion by Zzzen** [mobxjs/mobx#3433] [Q&A] [answered]
[observable.map](https://mobx.js.org/api.html#observablemap)
>Creates a new observable [ES6 Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) based on the provided initialMap. They are very useful if you don't want to react just to the change of a specific entry, but also to their addition and removal. Creating observable Maps is the recommended approach for creating dynamically keyed collections if you don't have [enabled Proxies](https://mobx.js.org/configuration.html#proxy-support).

IIUC, proxied observable can track addition and removal of entries.

**Comment by urugator** [comment, marked as answer]
Proxies are only relevant for plain object and array. The difference between observable Map and (proxied) observable object is the same as difference between non-observable [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and non-observable [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object). Both support dynamic keys.
Non-proxied observable object does not support addition/removal of keys using assigment/`delete` operator/`Reflect.*`/etc, but it's still possible using [collection utilities](https://mobx.js.org/api.html#collection-utilities-).
